docs/source/build-run-qualcomm-ai-engine-direct-backend.md

```diff
@@ -107,22 +109,38 @@ export PYTHONPATH=$EXECUTORCH_ROOT/..
 
 ## Build
 
-An example script for below building instructions is [here](https://github.com/pytorch/executorch/blob/main/backends/qualcomm/scripts/build.sh).
+An example script for the below building instructions is [here](https://github.com/pytorch/executorch/blob/main/backends/qualcomm/scripts/build.sh).
+We recommend to use the script because the ExecuTorch build-command can change from time to time.
+The above script is actively used. It is updated more frquently than this tutorial.
+An example usage is
+```bash
+cd $EXECUTORCH_ROOT
+./backends/qualcomm/scripts/build.sh
+# or
+./backends/qualcomm/scripts/build.sh --release
+```
 
 ### AOT (Ahead-of-time) components:
 
 Python APIs on x64 are required to compile models to Qualcomm AI Engine Direct binary.
 
 ```bash
 cd $EXECUTORCH_ROOT
-# Workaround for fbs files in exir/_serialize
-cp schema/program.fbs exir/_serialize/program.fbs
-cp schema/scalar_type.fbs exir/_serialize/scalar_type.fbs
-
 mkdir build_x86_64
 cd build_x86_64
```

Comment:
let's add them to git ignore?

Reply:
I'm fine with that... but how about using the name already in `.gitignore`?
```
cmake-out/
cmake-android-out/
cmake-out-android/
```
I can change `build_x86_64` to `cmake-out` and `build_android` to `cmake-android-out`. It seems more consistent with other backends.

Reply:
That sounds good